serialization/src/main/java/net/corda/v5/serialization/SerializedBytes.java

```diff
@@ -1,29 +1,22 @@
 package net.corda.v5.serialization;
 
 import net.corda.v5.base.annotations.CordaSerializable;
-import net.corda.v5.base.types.OpaqueBytes;
 import org.jetbrains.annotations.NotNull;
 
 /**
  * A type safe wrapper around a byte array that contains a serialised object.
  */
 @SuppressWarnings("unused")
 @CordaSerializable
-public final class SerializedBytes<T> extends OpaqueBytes {
-    /**
-     * Constructs a SerializedBytes holding the specified bytes.
-     * @param bytes Byte array holding serialized form of type T.
-     */
-    public SerializedBytes(@NotNull byte[] bytes) {
-        super(bytes);
-    }
+public interface SerializedBytes<T> {
+
+    @NotNull
+    byte[] getBytes();
 
     /**
      * Returns a text description of this SerializedBytes object
      * @return Text description of this SerializedBytes object.
      */
     @NotNull
-    public String getSummary() {
-        return "SerializedBytes(size = " + getSize() + ')';
-    }
+    String getSummary();
```

Comment:
Don't really see much use in this one, perhaps we should kill?

(Related to this one:) Perhaps we should be providing a more meaningful `toString` than the current `ByteSequence.toString` I think.

Reply:
(That was already there so maybe above comment could be done under different work/ PR)

Reply:
Let's do it now, it's not a big change and it would be better to do it all together.

Reply:
I think let's just remove then, doesn't offer much, seems more like unneeded overhead to the API to me

Reply:
Done.